prometheus/types.go

```diff
@@ -13,5 +13,6 @@ type Alert struct {
 type Scrape struct {
 	ScrapePort 	int `json:"scrapePort,string,omitempty"`
 	ServiceName string `json:"serviceName"`
+	ScrapeType string
```

Comment:
I guess that the primary use-case is to use `static_configs` through env. vars. In that case, there's no need to transform it from JSON requests. However, it would probably still be useful to add the option to accomplish `static_configs` also through a request. Adding `json:"scrapeType"` and a test should be enough. What do you think?